awx/ui_next/src/screens/Inventory/InventoryRelatedGroups/InventoryRelatedGroupList.jsx

```diff
@@ -0,0 +1,255 @@
+  const addFormUrl = `/home`;
+  const addButtonOptions = [];
+
+  if (canAdd) {
+    addButtonOptions.push(
+      {
+        onAdd: () => setIsModalOpen(true),
+        title: i18n._(t`Add existing group`),
+        label: i18n._(t`group`),
+        key: 'existing',
+      },
+      {
+        onAdd: () => history.push(addFormUrl),
+        title: i18n._(t`Add new group`),
+        label: i18n._(t`group`),
+        key: 'new',
+      }
+    );
+  }
+
+  const addButton = <AddDropdown key="add" dropdownItems={addButtonOptions} />;
+
+  return (
+    <>
+      <PaginatedDataList
+        contentError={contentError}
+        hasContentLoading={isLoading}
+        items={groups}
+        itemCount={itemCount}
+        pluralizedItemName={i18n._(t`Related Groups`)}
+        qsConfig={QS_CONFIG}
+        onRowClick={handleSelect}
+        toolbarSearchColumns={[
+          {
+            name: i18n._(t`Name`),
+            key: 'name__icontains',
+            isDefault: true,
+          },
+          {
+            name: i18n._(t`Created By (Username)`),
+            key: 'created_by__username__icontains',
+          },
+          {
+            name: i18n._(t`Modified By (Username)`),
+            key: 'modified_by__username__icontains',
+          },
+        ]}
+        toolbarSortColumns={[
+          {
+            name: i18n._(t`Name`),
+            key: 'name',
+          },
+        ]}
+        toolbarSearchableKeys={searchableKeys}
+        toolbarRelatedSearchableKeys={relatedSearchableKeys}
+        renderToolbar={props => (
+          <DataListToolbar
+            {...props}
+            showSelectAll
+            isAllSelected={isAllSelected}
+            onSelectAll={isSelected =>
+              setSelected(isSelected ? [...groups] : [])
+            }
+            qsConfig={QS_CONFIG}
+            additionalControls={[
+              ...(canAdd ? [addButton] : []),
+              <Kebabified>
+                {({ isKebabified }) =>
+                  isKebabified ? (
+                    <AdHocCommandsButton
+                      adHocItems={selected}
+                      apiModule={InventoriesAPI}
+                      itemId={parseInt(inventoryId, 10)}
+                    >
+                      {({ openAdHocCommands, isDisabled }) => (
+                        <DropdownItem
+                          key="run command"
+                          onClick={openAdHocCommands}
+                          isDisabled={itemCount === 0 || isDisabled}
+                        >
+                          {i18n._(t`Run command`)}
+                        </DropdownItem>
+                      )}
+                    </AdHocCommandsButton>
+                  ) : (
+                    <ToolbarItem>
+                      <Tooltip
+                        content={i18n._(
+                          t`Select an inventory source by clicking the check box beside it. The inventory source can be a single host or a selection of multiple hosts.`
```

Comment:
I really wish our code formatter would handle these strings, but it doesn't : \ 

I'm not picky about these in general, but can you please format help strings so that the entire message can be read in a code review window without side scrolling? ~100 or ~120 chars is generally a good number to aim for.

Reply:
For example, I reviewed this community pr the other day (proof-reading for typos ;) )


![lines](https://user-images.githubusercontent.com/9753817/94737294-9dc7c180-033b-11eb-8384-f779725a72d9.gif)